doc/api/http2.md

```diff
@@ -2488,6 +2499,30 @@ server.on('stream', (stream, headers) => {
 });
 ```
 
+<a id="http2-sensitive-headers"></a>
+#### Sensitive headers
+
+HTTP2 headers can be marked as sensitive, which means that the HTTP/2
+header compression algorithm will never index them. This can make sense for
+header values with low entropy and that may be considered valuable to an
+attacker, for example `Cookie` or `Authorization`. To achieve this, add
+the header name to the `[http2.sensitiveHeaders]` property as an array:
+
+```js
+const headers = {
+  ':status': '200',
+  'content-type': 'text-plain',
+  'cookie': 'some-cookie',
+  'other-sensitive-header': 'very secret data',
+  [http2.sensitiveHeaders]: ['cookie', 'other-sensitive-header']
+};
+
+stream.respond(headers);
+```
+
+For some headers, such as `Authorization` and short `Cookie` headers,
+this flag is set automatically.
+
```

Comment:
Shouldn't we also mention that sensitive headers' names will be present under `headers[http2.sensitiveHeaders]` for incoming headers? And that it may contain more elements than the original sensitive-headers-array as per https://github.com/nodejs/node/pull/34145#issuecomment-652105262.

Reply:
Sure, done :+1: